commands/priority.go

```diff
@@ -30,3 +33,17 @@ func getGatewayURL(argumentURL, defaultURL, yamlURL, environmentURL string) stri
 
 	return gatewayURL
 }
+
+func getTemplateURL(argumentURL, defaultURL, environmentURL string) string {
```

Comment:
I can see how we have the args in this order, but I wonder whether the order should, at least initially, reflect the priority order.  I'd find the calls to the method easier to read.